packages/SwingSet/src/lib/message.js

```diff
@@ -1,6 +1,12 @@
 import { assert, Fail } from '@endo/errors';
 import { insistCapData } from './capdata.js';
 
+/**
+ * @typedef { import('@agoric/swingset-liveslots').VatSyscallObject } VatSyscallObject
+ * @typedef { import('@agoric/swingset-liveslots').VatSyscallResult } VatSyscallResult
+ * @typedef { import('@agoric/swingset-liveslots').VatDeliveryResult } VatDeliveryResult
+ */
+
```

Comment:
It is better to alias imported types than to create new types which are just copies of imported ones.

```suggestion
/**
 * @import { VatSyscallObject, VatSyscallResult, VatDeliveryResult }
 *     from '@agoric/swingset-liveslots';
 */

```